instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteRefreshTest.kt

```diff
@@ -241,8 +241,8 @@ class RouteRefreshTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.ja
             alternativesMetadata.navigationRoute.id
         )
         assertNotEquals(alternativesMetadataLegacy, alternativesMetadata)
-        assertEquals(266.0, alternativesMetadataLegacy.infoFromStartOfPrimary.duration, 1.0)
-        assertEquals(275.0, alternativesMetadata.infoFromStartOfPrimary.duration, 1.0)
+        assertEquals(267.9, alternativesMetadataLegacy.infoFromStartOfPrimary.duration, 1.0)
+        assertEquals(277.0, alternativesMetadata.infoFromStartOfPrimary.duration, 1.0)
```

Comment:
Noting that I had to change the values here. Is that expected? @RingerJK @LukasPaczos

Reply:
Probably yes due to

> Fixed calculation of alternative route info (duration and distance)

cc @DenisPryt

Reply:
In this case we should add it to our changelog as well.

Reply:
Great point. Just added it 👍